pages/form-builder/responses/[[...params]].tsx

```diff
@@ -90,58 +187,29 @@ const Responses: NextPageWithLayout<ResponsesProps> = ({ vaultSubmissions }: Res
 
         {isAuthenticated && (
           <>
-            <div className="border-2 border-solid border-black">
-              <table>
-                <thead>
-                  <tr>
-                    <th>Response ID</th>
-                    <th>Status</th>
-                    <th>Download Response</th>
-                    <th>Last Downloaded By</th>
-                    <th>Confirm Receipt</th>
-                    <th>Removal</th>
-                  </tr>
-                </thead>
-                <tbody>
-                  <>
-                    {vaultSubmissions?.length > 0 ? (
-                      vaultSubmissions.map((submission, index) => (
-                        <tr key={index}>
-                          <td>{submission.name}</td>
-                          <td>{submission.status}</td>
-                          <td></td>
-                          <td></td>
-                          <td>
-                            {submission.status === "Confirmed" ? "Confirmed" : "Confirm By XXXX"}
-                          </td>
-                          <td>Not Set</td>
-                        </tr>
-                      ))
-                    ) : (
-                      <>
-                        <tr key="1">
-                          <td></td>
-                          <td></td>
-                          <td></td>
-                          <td></td>
-                          <td></td>
-                          <td>Not Set</td>
-                        </tr>
-                        <tr key="2">
-                          <td></td>
-                          <td></td>
-                          <td></td>
-                          <td></td>
-                          <td></td>
-                          <td>Not Set</td>
-                        </tr>
-                      </>
-                    )}
-                  </>
-                </tbody>
-              </table>
+            <div>
+              {vaultSubmissions?.length > 0 && (
```

Comment:
`vaultSubmissions` will always be defined. No need for the optional chaining parameter `.?`
```suggestion
              {vaultSubmissions.length > 0 && (
```